### PR TITLE
Aquastream XT sensor support patch for hwmon

### PR DESCRIPTION
**Pull request by aleksamagicka**
Posting the patch for adding Aquastream XT sensor support to the kernel. @leoratte, please review, thanks.

### CODE REVIEW

File: docs/aquacomputer_d5next.rst [hunk 1]
```diff
@@ -56,6 +57,9 @@ The High Flow Next exposes +5V voltages, water quality, conductivity and flow re
 A temperature sensor can be connected to it, in which case it provides its reading
 and an estimation of the dissipated/absorbed power in the liquid cooling loop.
 
+The Aquastream XT pump exposes temperature readings for the coolant, external sensor
+and fan IC. It also exposes pump and fan voltages, as well as pump current.
```
Comment (leoratte) [inline docs/aquacomputer_d5next.rst:61]: Pump and fan RPM are also supported.

File: aquacomputer_d5next.c [hunk 2]
```diff
@@ -231,6 +236,26 @@ static u16 quadro_ctrl_fan_offsets[] = { 0x37, 0x8c, 0xe1, 0x136 }; /* Fan speed
...
+/* Specs of the Aquastream XT pump */
+#define AQUASTREAMXT_SERIAL_START		0x3a
+#define AQUASTREAMXT_FIRMWARE_VERSION		0x32
+#define AQUASTREAMXT_NUM_FANS			2
+#define AQUASTREAMXT_NUM_SENSORS		3
+#define AQUASTREAMXT_FAN_STOPPED		0x4
+#define AQUASTREAMXT_PUMP_CONVERSION_CONST	45000000
+#define AQUASTREAMXT_FAN_CONVERSION_CONST	5646000
+#define AQUASTREAMXT_PUMP_MIN_RPM		3000
+#define AQUASTREAMXT_PUMP_MAX_RPM		6000
```
Comment (leoratte) [inline aquacomputer_d5next.c:248]: These two are currently unused so remove them.

**Review by leoratte — CHANGES_REQUESTED** [review]
Looks good so far, only two small comments.
I don't have the Aquastream XT hooked up currently, so I'll test this in the next 2 or 3 weeks.

File: docs/aquacomputer_d5next.rst (hunk 1 above)
Comment (aleksamagicka) [reply docs/aquacomputer_d5next.rst]: Thanks, fixed this and the other comment.

**Review by leoratte — APPROVED** [review]
Looks good now. I've also tested this and everything works as expected.

**Comment by aleksamagicka** [comment]
Excellent, thank you!

**Comment by aleksamagicka** [comment]
Force pushing the commit with updated description, no other changes made. Will send to LKML soon.

**Comment by aleksamagicka** [comment]
@leoratte, just sent this and realized I didn't include you in the CC - sorry! Patch is [here](https://patchwork.kernel.org/project/linux-hwmon/patch/20230416181702.9892-1-savicaleksa83@gmail.com/).

**Comment by aleksamagicka** [comment]
This got in, merging.

**Pull request merged**